templates/base.html

```diff
@@ -59,5 +71,28 @@
     <script type="text/javascript" src="{{ url_for('static', filename='js/index.js') }}"></script>
     <script type="text/javascript" src="https://s3.amazonaws.com/downloads.mailchimp.com/js/signup-forms/popup/embed.js" data-dojo-config="usePlainJson: true, isDebug: false"></script>
     <script type="text/javascript"></script>
+    <script type="text/javascript" language="javascript">
+       $(document).ready(function() {
+            function numberWithCommas(x) {
+              return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+            }
+            $.getJSON("http://api.indiegogo.com/1/campaigns/14952534#.json?api_token=060335a95d69355cf5e3bef25c36d1be61c8389c3d32c54b402fd5034304df9d", function(gogodata) {
```

Comment:
@ErinOC possible security issue with the token exposed client-side? What about rendering on server.